### DISCUSSION TITLE
how to remove repo not belonging to me on dashboard

### DISCUSSION THREAD

**Discussion by ghost** [community/community#126493] [New to GitHub]
![2024-05-30 11 02 51](https://github.com/community/community/assets/6504074/d46e2980-ec84-40cf-ba9e-6c6998215b95)
 
warpdotdev/Warp, I didn't watch,star,fork the repo, why it just appears there

**Comment by pedrolemos4** [comment]
I don't think that it is possible to remove it...
GitHub currently does not have that feature. 
You can check this thread https://github.com/orgs/community/discussions/40959

**Comment by Aryosetowmn** [comment]
I guess there is no way to do that. In my opinion, once you have done any things like submit an issue, fork or contribute or any things. It save there as history. It doesn't matter if you close your issue on that repo. These are autogenerated based on your activity 👍.
